### PR TITLE
DOCSP-45003: Specify Fields to Include

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-45003>

### Staging Links

<li><a href=https://deploy-preview-291--mongodb-docs-csharp.netlify.app/fundamentals/crud/read-operations/project>fundamentals/crud/read-operations/project</a></li><li><a href=https://deploy-preview-291--mongodb-docs-csharp.netlify.app/fundamentals/crud/read-operations>fundamentals/crud/read-operations</a></li>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/project.txt [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+To specify the fields to include from the result, chain the ``Project()`` method
+to the ``Find()`` method. When calling the ``Project()`` method, you must pass in the
+projection definition as a parameter. You can construct a projection definition by using
+the ``Builders<T>.Projection.Include()`` method and passing in the field name to include
+as a parameter. This method can be chained to include multiple fields in the projection.
+
+The following example uses the ``Find()`` method to find all restaurants in which the ``name``
+field value is ``"Emerald Pub"``. Then, the code calls the ``Project()``
+method to instruct the find operation to include the ``name`` and ``address`` fields
```
Comment (stephmarie17) [inline source/fundamentals/crud/read-operations/project.txt:67]: [q] Should this be updated to match the code snippet?
```suggestion
method to instruct the find operation to include the ``name`` and ``cuisine`` fields
```

File: source/fundamentals/crud/read-operations/project.txt [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+   .. output::
+      :visible: false
+
+      { "cuisine" : "American", "name" : "Emerald Pub" }
+      { "cuisine" : "American", "name" : "Emerald Pub" }
+
+Specify Fields to Exclude
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To specify the fields to include from the result, chain the ``Project()`` method
```
Comment (stephmarie17) [inline source/fundamentals/crud/read-operations/project.txt:114]: Typo: 
```suggestion
To specify the fields to exclude from the result, chain the ``Project()`` method
```

**Review by stephmarie17 — APPROVED** [review]
LGTM with a few small comments!

File: source/includes/fundamentals/code-examples/Project.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+public class Address
+{
+    public string Building { get; set; }
+
+    [BsonElement("coord")]
+    public double[] Coordinates { get; set; }
+
+    public string Street { get; set; }
```
Comment (rstam) [inline source/includes/fundamentals/code-examples/Project.cs:86]: Did you mean to omit `[BsonElement]` here?

File: source/includes/fundamentals/code-examples/Project.cs [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    [BsonElement("cuisine")]
+    public string? Cuisine { get; set; }
+
+    [BsonElement("address")]
+    public Address? Address { get; set; }
+}
+
+public class Address
+{
+    public string Building { get; set; }
```
Comment (rstam) [inline source/includes/fundamentals/code-examples/Project.cs:81]: Did you mean to omit [BsonElement] here?

File: source/fundamentals/crud/read-operations/project.txt [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+ 
+.. meta::
+   :keywords: read, filter, project, select
+
+Overview
+--------
+
+In this guide, you can learn how to specify which fields to return from a read
+operation by using a **projection**. A projection is a document that specifies
+which fields MongoDB returns from a query.
```
Comment (rstam) [inline source/fundamentals/crud/read-operations/project.txt:25]: Something to consider is that projections don't play nicely with POCOs.

When you only return some of the fields from the server your POCO is incomplete. Depending on how the serialization is done either the missing fields will be `null` (or other suitable default value) in your POCO result or perhaps even an exception might be thrown for missing fields.

**Review by rstam — CHANGES_REQUESTED** [review]

File: source/fundamentals/crud/read-operations/project.txt (hunk 5 above)
Comment (mcmorisi) [reply source/fundamentals/crud/read-operations/project.txt:25]: I'll update this page to avoid using POCOs

**Pull request merged**